src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs

```diff
@@ -231,7 +231,7 @@ public static int LastIndexOf(ref byte searchSpace, int searchSpaceLength, ref b
             }
             return -1;
 
-        // Based on http://0x80.pl/articles/simd-strfind.html#algorithm-1-generic-simd "Algorithm 1: Generic SIMD" by Wojciech Muła
+        // Based on http://0x80.pl/articles/simd-strfind.html#algorithm-1-generic-simd "Algorithm 1: Generic SIMD" by Wojciech Mu\u0142a
```

Comment:
😄

Reply:
Ah 😅
```
$ git grep Wojciech
THIRD-PARTY-NOTICES.TXT:Copyright (c) 2015-2017, Wojciech Mula
THIRD-PARTY-NOTICES.TXT:License for sse4-strstr (https://github.com/WojciechMula/sse4-strstr)
THIRD-PARTY-NOTICES.TXT:   Copyright (c) 2008-2016, Wojciech Muła
src/installer/pkg/THIRD-PARTY-NOTICES.TXT:Copyright (c) 2015-2017, Wojciech Mula
src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs:            // Based on http://0x80.pl/articles/simd-strfind.html#algorithm-1-generic-simd "Algorithm 1: Generic SIMD" by Wojciech Mu\u0142a
src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs:        // Based on http://0x80.pl/articles/simd-strfind.html#algorithm-1-generic-simd "Algorithm 1: Generic SIMD" by Wojciech Mu\u0142a
src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Char.cs:            // Based on http://0x80.pl/articles/simd-strfind.html#algorithm-1-generic-simd "Algorithm 1: Generic SIMD" by Wojciech Mu\u0142a
src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Char.cs:            // Based on http://0x80.pl/articles/simd-strfind.html#algorithm-1-generic-simd "Algorithm 1: Generic SIMD" by Wojciech Mu\u0142a
```
I'll streamline it to `src/installer` one with smalll `L`.

Same thing happened with `Hernàndez`. In some files, it was `Hernandez`, in others it was `Hern?ndez`, so I streamlined it to ASCII `a`.

Reply:
Do we want to make zero exceptions and convert them all?

Reply:
> Literal non-ASCII characters occasionally get garbled by a tool or editor.

What common tools still struggle with this? It seems odd that we'd be stuck with garbling people's names manually so that some tool doesn't get a chance to.

Reply:
Yeah the justification for this coding style rule was something like: a test string in a unit test gets translated to ASCII and nobody notices. I believe that happened, but I could never find it now. I feel uneasy changing people's actual names in comments.

Reply:
How about just ASCIIfying the cases of actual characters/strings or comments about them? Unless we add some CI validation, we will never be all clean.

Reply:
We had `Hernàndez`, `Hernandez` (all ASCII) and `Hern?ndez` (incorrect) for the same author which I've replaced with `Hernandez` (all ASCII); not introducing any "new" variant here.

`Wojciech Muła` uses ASCII form like this: https://github.com/WojciechMula (small `L`), so I don't think it is wrong.

Those are the only two names in this 520 lines PR, rest of the source code is already using escaped sequences.